ProjectSettings/ProjectSettings.asset

```diff
@@ -8,10 +8,10 @@ PlayerSettings:
   AndroidProfiler: 0
   AndroidFilterTouchesWhenObscured: 0
   AndroidEnableSustainedPerformanceMode: 0
-  defaultScreenOrientation: 3
+  defaultScreenOrientation: 4
```

Comment:
Are all of these changes intended?

Reply:
Yes, iOS has a higher refresh rate capability in portrait